docs/jetstream/operations.md

```diff
@@ -0,0 +1,129 @@
+---
+id: operations
+title: Jetstream Architecture and Operations
+---
+
+**[Jetstream]** is part of the Cirrus ecosystem and depends on some external services.
+
+<img src="/experimenter-docs/img/jetstream/cirrus.png" alt="Cirrus architecture" className="img-xl" />
+
+Jetstream is [scheduled to run in Airflow](https://github.com/mozilla/telemetry-airflow/blob/e5de501d8063cc366e9bb546135f3866136cb47d/dags/jetstream.py#L22) daily. The daily runs will analyze all experiments that are currently active or just ended the day before and write metrics, statistics and errors for each experiment to BigQuery. Active V1 experiments and V6 experiments (Nimbus experiments) are retrieved from the [Experimenter API](https://experimenter.services.mozilla.com/api/v1/experiments/).
+
+Jetstream also fetches custom experiment and outcome configs from [jetstream-config](https://github.com/mozilla/jetstream-config) for analysis. When a new custom config gets merged into jetstream-config, the CI will trigger Jetstream to re-run all analyses for the experiment affected by the config. CircleCI will report on the status of the analysis run and link to the Cloud Logging logs.
```

Comment:
Writing this, I started to wonder if we should also write errors that can occur on a rerun to BigQuery?

Reply:
Yeah probably :D

I was musing earlier that it'd be nice to use structured logging to trace execution times for different components; I don't know that there's a good ROI but it'd be interesting to know how slow some of our stats are.

Reply:
Interesting, that might indeed be nice to have!

Reply:
Sorry, total non-sequitur here!